tests/opentelemetry-docker-tests/tests/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+
+services:
+  otmongo:
+    ports:
+      - "27017:27017"
+    image: mongo:latest
+  otmysql:
+    ports:
+      - "3306:3306"
+    image: mysql:latest
+    restart: always
+    environment:
+      MYSQL_USER: testuser
+      MYSQL_PASSWORD: testpassword
+      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+      MYSQL_DATABASE: opentelemetry-tests
+  otpostgres:
+    image: postgres
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: testuser
+      POSTGRES_PASSWORD: testpassword
+      POSTGRES_DB: opentelemetry-tests
+  otredis:
+      image: redis:4.0-alpine
+      ports:
+        - "127.0.0.1:6379:6379"
+  otjaeger:
+    image: jaegertracing/all-in-one:1.8
+    environment:
+      COLLECTOR_ZIPKIN_HTTP_PORT: "9411"
+    ports:
+        - "5775:5775/udp"
+        - "6831:6831/udp"
+        - "6832:6832/udp"
+        - "5778:5778"
+        - "16686:16686"
+        - "14268:14268"
+        - "9411:9411"
+  otopencensus:
```

Comment:
The otopencensus tests should remain in the core repo as the opencensus exporter will remain there.

Reply:
Sounds good! I've removed all the OpenCensus Tests.